### PR TITLE
Prepare WorkspaceStorage binding

### PR DESCRIPTION
**Pull request by TimeEngineer**

### CODE REVIEW

**Review by FirelightFlagboy — COMMENTED** [review]
python

File: parsec/core/sync_monitor.py [hunk 1]
```diff
@@ -135,7 +135,9 @@ async def _load_changes(self) -> bool:
             changes = rep["changes"]
 
         # 2) Store new checkpoint and changes
-        await self._get_local_storage().update_realm_checkpoint(new_checkpoint, changes)
+        await self._get_local_storage().update_realm_checkpoint(
+            new_checkpoint, {EntryID.from_hex(name.hex): val for name, val in changes.items()}
```
Comment (vxgmichel) [inline parsec/core/sync_monitor.py:139]: Good catch :+1:

File: parsec/core/fs/storage/workspace_storage.py [hunk 2]
```diff
@@ -419,6 +426,12 @@ async def mark_prevent_sync_pattern_fully_applied(self, pattern: Pattern[str]) -
         await self.manifest_storage.mark_prevent_sync_pattern_fully_applied(pattern)
         await self._load_prevent_sync_pattern()
 
+    async def chunk_storage_get_local_chunk_ids(self, chunk_id: List[ChunkID]) -> List[ChunkID]:
+        return await self.chunk_storage.get_local_chunk_ids(chunk_id)
+
+    async def block_storage_get_local_chunk_ids(self, chunk_id: List[ChunkID]) -> List[ChunkID]:
+        return await self.block_storage.get_local_chunk_ids(chunk_id)
```
Comment (vxgmichel) [inline parsec/core/fs/storage/workspace_storage.py:433]: Maybe those methods should be called `get_local_chunk_ids` and `get_local_block_ids`?

**Review by vxgmichel — APPROVED** [review]
LGTM :+1:

**Review by touilleMan — APPROVED** [review]

**Pull request merged**